src/utils/GetSkillName.js

```diff
@@ -0,0 +1,9 @@
+const GetSkillName = skillTag => {
```

Comment:
Name the variable as `getSkillNameFromSkillTag`

Reply:
Done Sir